style.css

```diff
@@ -0,0 +1,91 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+}
+
+h1 {
+  color: #172b4d;
+  font-size: 49px;
+  line-height: 52px;
+  text-align: left;
+  width: 327px;
```

Comment:
- [ ] [OPTIONAL] Consider avoiding using static width for your section as it will not auto-resize on mobile with larger or smaller screen the predefined size. Consider using "%"